Source/Scene/Points3DTileContent.js

```diff
@@ -403,10 +562,21 @@ define([
             vs += '    vec3 position = a_position; \n';
         }
 
+        // TODO : only for testing
```

Comment:
I'm going to remove it. This was mainly for anyone reviewing the PR to see how it works.